charts/internal/calico/templates/typha/deployment-calico-typha.yaml

```diff
@@ -85,6 +85,8 @@ spec:
           name: metrics
           protocol: TCP
         {{-  end }}
+        securityContext:
+          allowPrivilegeEscalation: false
```

Comment:
```suggestion
```
This container already has a `securityContext` with `allowPrivilegeEscalation: false`